jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/comments/CaseCommentItemView.java

```diff
@@ -250,6 +254,22 @@ public void execute() {
                     }
                 });
             }
+            addAction(new CaseCommentsPresenter.CaseCommentAction() {
+                @Override
+                public String label() {
+                    return translationService.format(DELETE);
+                }
+
+                @Override
+                public void execute() {
+                    confirmPopup.show(translationService.format(DELETE_COMMENT),
```

Comment:
For the pop up _Delete Comment_ is fine but as for the button, can we use just _Delete_ instead?